API/API/Program.cs

```diff
@@ -13,16 +11,22 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
-var connectionString = builder.Configuration.GetConnectionString("PostgreSQLConnection");
+//Conection to data base
+string? connectionString = builder.Configuration.GetConnectionString("PostgreSQLConnection");
```

Comment:
To have the code clear you can create a static method like "DependencyInjectionConfigure.ConfigureService"

Reply:
Done